### PR TITLE
Fixes the model parser and tests in show version

### PR DESCRIPTION
**Pull request by LukeDRussell**
Signed-off-by: Luke Russell <lukedrussell+git@outlook.com>

### CODE REVIEW

**Comment by pabelanger** [comment]
recheck

**Comment by pabelanger** [comment]
Okay, now we are properly gating on parsing tests. you can now iterate properly on your patch

**Comment by LukeDRussell** [comment]
recheck

**Comment by LukeDRussell** [comment]
Done.

File: tests/parser_templates/cli/show_interfaces/main.yaml [hunk 1]
```diff
@@ -18,9 +18,9 @@
       - interfaces['GigabitEthernet0/0/1']['admin-status'] == 'enabled'
       - interfaces['Tunnel0']['admin-status'] == 'enabled'
       - interfaces['Tunnel0']['oper-status'] == 'down'
-      - interfaces['GigabitEthernet0']['counters']['in-broadcast-pkts'] == '4891144'
+      - interfaces['GigabitEthernet0']['counters']['in-broadcast-pkts'] | int == 4891144
```
Comment (pabelanger) [inline tests/parser_templates/cli/show_interfaces/main.yaml]: There shouldn't be a need to cast here, if you revert this change, it will work.

File: tests/parser_templates/cli/show_interfaces/main.yaml [hunk 2]
```diff
@@ -18,9 +18,9 @@
...
       - interfaces['GigabitEthernet0']['counters']['in-multicast-pkts'] == None
-      - interfaces['Virtual-Access1']['mtu'] == '1500'
+      - interfaces['Virtual-Access1']['mtu'] | int == 1500
```
Comment (pabelanger) [inline tests/parser_templates/cli/show_interfaces/main.yaml]: same comment

**Review by pabelanger — CHANGES_REQUESTED** [review]
Thanks, if you want to address comments and rebase against current master, into single commit, should make for easier reviewing.

File: tests/parser_templates/cli/show_interfaces/main.yaml (hunk 2 above)
Comment (LukeDRussell) [reply tests/parser_templates/cli/show_interfaces/main.yaml]: I think the output of these parsers should be integers for straight numbers. 

This may be a problem with `command_parser`, as I haven't been able to get it to return int types, even when using jinja2 filters.

I'll revert these for now to get the other stuff in, but these should be moved to integers at some point.

File: tests/parser_templates/cli/show_version/16.6.4.txt [hunk 3]
```diff
@@ -64,19 +64,18 @@ Technology Package License Information:
 -----------------------------------------------------------------
 Technology    Technology-package           Technology-package
               Current       Type           Next reboot
-              ------------------------------------------------------------------
```
Comment (pabelanger) [inline tests/parser_templates/cli/show_version/16.6.4.txt:67]: what's the reason for the formatting change here? I am guessing copypasta issue on original commit?

File: tests/parser_templates/cli/show_version/main.yaml [hunk 4]
```diff
@@ -1,4 +1,22 @@
 ---
+### IOS 15.1(4)M4
```
Comment (pabelanger) [inline tests/parser_templates/cli/show_version/main.yaml:2]: do you mind moving this back to original location, this helps keep the diff smaller and easier to review.

File: tests/parser_templates/cli/show_version/16.6.4.txt (hunk 3 above)
Comment (LukeDRussell) [reply tests/parser_templates/cli/show_version/16.6.4.txt:67]: Yes, it was a copy paste problem.

File: tests/parser_templates/cli/show_version/main.yaml (hunk 4 above)
Comment (LukeDRussell) [reply tests/parser_templates/cli/show_version/main.yaml:2]: The tests are in ascending order now, which I think is more important than diff size. Hopefully there will be more test samples from others in the future.
